### DISCUSSION TITLE
Is it possible to make a tracked vehicle model in carla?

### DISCUSSION THREAD

**Discussion by KimBooKun** [carla-simulator/carla#7676] [Q&A]
I wanna make tracked vehicle model (like panzer) in carla simulator.

I found it is possible to connect carla and chrono by using 'enable_chrono_physics' method in carla documentation.
and also, I found a tracked vehicle model's json file in project chrono.
This method uses just 3 inputs about vehicle's model. (1 vehicle.json, 2. powertrain.json, 3. tire.json)

There is only one example about wheeled vehicle by using tire.json in carla documentation.
But I have to use track assembly json file, not tire.json

I wanna ask if it 's possible to make a Tracked Vehicle in carla simulator?
Or, is there any other possible way to make a tracked vehicle model in carla?

Please help me. Thank you for your kind help in advance. :)

**Comment by xavisolesoft** [comment]
Ye is it possible but modifying the CARLA C++ code.
Please, takes as example the implementation of the C++ birdge between CARLA and Chrono:
https://github.com/carla-simulator/carla/blob/dev/Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Vehicle/MovementComponents/ChronoMovementComponent.cpp#L0-L1
